recipes/noqt/meta.yaml

```diff
@@ -0,0 +1,66 @@
+{% set version = "1.0" %}
+
+package:
+  name: noqt
+  version: {{ version }}
+
+
+build:
+  number: 0
+  noarch: generic
+
+requirements:
+  run:
+    - {{ pin_subpackage('noqt4', exact=True) }}
+    - {{ pin_subpackage('noqt5', exact=True) }}
+    - {{ pin_subpackage('noqt6', exact=True) }}
+test:
+  commands:
+    - echo "no tests for this package"
+
+outputs:
+  - name: noqt
+  - name: noqt4
+    build:
+      noarch: generic
+    requirements:
+      run_constrained:
+        # qt4 only ever had a "mega qt" package called qt
+        - qt <0.a0|>=5.a0
+    test:
+      commands:
+        - echo "no tests for this package"
```

Comment:
I would skip `noqt4` completely. qt4 is so old (the last [build](https://anaconda.org/conda-forge/qt/files?version=4.8.7) was >5 years ago) that it's essentially uninstallable anyway. 

With `noqt5` requiring both `qt <0.0a0` and `qt-main <0.0a0`, that already encompasses all remnants of qt4 that might exist.

Reply:
ok. i removed `noqt4`.